ci/Jenkinsfile.android

```diff
@@ -90,4 +92,7 @@ pipeline {
       }
     }
   }
+  post {
+    failure { script { cmn.gitHubNotifyPRFail() } }
```

Comment:
Does this ever get called when we're not building for a PR branch (i.e. `env.CHANGE_ID` doesn't have a value)?

Reply:
Good catch, yeah, I just caught that, so I added it as an argument the `buildBranch` method. Check it out.